tests/models/flava/test_modeling_flava.py

```diff
@@ -746,17 +746,31 @@ class FlavaModelTester:
     def __init__(
         self,
         parent,
+        text_kwargs=None,
+        image_kwargs=None,
+        multimodal_kwargs=None,
+        image_codebook_kwargs=None,
         is_training=True,
         hidden_size=32,
         projection_dim=32,
         initializer_range=0.02,
         layer_norm_eps=1e-12,
     ):
+
+        if text_kwargs is None:
+            text_kwargs = {}
+        if image_kwargs is None:
+            image_kwargs = {}
+        if multimodal_kwargs is None:
+            multimodal_kwargs = {}
+        if image_codebook_kwargs is None:
+            image_codebook_kwargs = {}
+
```

Comment:
cc @sgugger 

`Flava` has 4 sub model testers. 
I also keep `image_kwargs` to align with `image_model_tester`, as well as the `image_config` attribute in `FlavaConfig`